autotest/t069_test_vtkexportmodel.py

```diff
@@ -273,8 +287,12 @@ def test_vtk_export_disu1_grid():
     ncpl = np.array(3 * [len(iverts)])
     nnodes = np.sum(ncpl)
 
-    top = np.ones((nnodes),)
-    botm = np.ones((nnodes),)
+    top = np.ones(
+        (nnodes),
+    )
+    botm = np.ones(
+        (nnodes),
+    )
```

Comment:
This seems to be black overreach, probably because (nnodes) is not a tuple.  I suspect that if we use:

```
top = np.ones(nnodes)
bot = np.ones(nnodes)
```

that this would prevent black from turning 2 lines into 6 lines.